index.js

```diff
@@ -1576,3 +1610,5 @@ ShareDbMongo.parseQueryError = function(err) {
   err.code = 5104;
   return err;
 };
+
+require('./src/middleware')(ShareDbMongo);
```

Comment:
Yup - that is a great suggestion thanks! I was mostly just trying to avoid adding more complexity to `index.js`

The updates were made in https://github.com/share/sharedb-mongo/pull/105/commits/3ed9f54e678cc0e10d552167137f36e22068129e